src/test/utility.hpp

```diff
@@ -131,7 +131,8 @@ run_command_output run_command(std::string command) {
   using boost::posix_time::microsec_clock;
   using boost::posix_time::ptime;
   FILE *in;
-  in = popen(command.c_str(), "r");
+  std::string command_plus = command + " 2>&1";  // put stderr to stdout
```

Comment:
Why does this pipe stderr to stdout?

Reply:
see https://stackoverflow.com/questions/6900577/c-popen-wont-catch-stderr
CLI11 lib writes arg parse errors to stderr; in order to check error message, we need stderr output as well.